metrics/bertscore/bertscore.py

```diff
@@ -105,7 +109,14 @@ def _compute(
         if num_layers is None:
             num_layers = bert_score.utils.model2layers[model_type]
 
-        hashcode = bert_score.utils.get_hash(model_type, num_layers, idf, rescale_with_baseline)
+        hashcode = bert_score.utils.get_hash(
+            model_type,
+            num_layers,
+            idf,
+            rescale_with_baseline,
+            baseline_path is not None,
```

Comment:
Could you use keyword arguments here to make it more robust to future changes ?

Reply:
Okay, done :)